apps/vaporgui/PCopyRegionAnnotationWidget.cpp

```diff
@@ -0,0 +1,10 @@
+#include "PCopyRegionAnnotationWidget.h"
+#include "CopyRegionAnnotationWidget.h"
+#include <vapor/ControlExecutive.h>
+#include <vapor/RenderParams.h>
+
+using namespace VAPoR;
+
+PCopyRegionAnnotationWidget::PCopyRegionAnnotationWidget(ControlExec *ce) : PWidget("", _widget = new CopyRegionAnnotationWidget(ce)) {}
+
+void PCopyRegionAnnotationWidget::updateGUI() const { _widget->Update(getParamsMgr()); }
```

Comment:
Why would the update method need the ParamsMgr as a parameter if the control exec already has a copy?

Reply:
Fair.  It's now accessed through the controlExec.